src/main/java/org/opensearch/security/dlic/rest/api/MultiTenancyConfigApiAction.java

```diff
@@ -33,6 +34,7 @@
 import org.opensearch.security.dlic.rest.validation.RequestContentValidator;
 import org.opensearch.security.dlic.rest.validation.RequestContentValidator.DataType;
 import org.opensearch.security.securityconf.impl.CType;
+import org.opensearch.security.securityconf.impl.DashboardSignInOption;
```

Comment:
wdyt about renaming this class to `DashboardsConfigApiAction`?

Reply:
This class is just an enum to match Sign In values, so I think `DashboardsConfigApiAction` is not an appropriate name because it doesn't do anything else.

Reply:
Woops, I added my comment on this line because it was the closest to the top of the changes in this file. The comment is not refereing to `DashboardsSigninOptions`, but rather `MultiTenancyConfigApiAction` (the name of the class of the file). wdyt about change the name from `MultiTenancyConfigApiAction` to `DashboardsConfigApiAction`?

Reply:
I may file an issue to make this class more generic in general which would need to be done in tandem with the frontend since the routes in this class begin with `/tenancy/...`.

Reply:
@cwperks, I agree about the class name change and as you mentioned, it can be another issue so it aligns with the work that needs to be done.